src/CBT.NuGet/Internal/AggregatePackage.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading;
+
+namespace CBT.NuGet.Internal
+{
+    internal sealed class AggregatePackage
+    {
+        public enum AggregateOption
+        {
+            Add,
+            Remove,
+        }
+
+        public AggregatePackage(string outPropertyId, List<KeyValuePair<AggregateOption, string>> packagesToAggregate, string destinationRoot)
```

Comment:
Is `packagesToAggregate` a `List<T>` because it can contain duplicates or can it be an `IDictionary<TKey, TValue>`?